src/main.js

```diff
@@ -1,6 +1,27 @@
-import './style.css';
-
 document.querySelector('#app').innerHTML = `
   <h1>Hello Vite!</h1>
   <a href="https://vitejs.dev/guide/features.html" target="_blank">Documentation</a>
 `;
+
+
+// DATA FROM 250 BEST MOVIES
+fetch("https://imdb-api.com/en/API/Top250Movies/k_yju1poqs/")
+    .then(result => result.json())
+    .then(({items}) => {                // <---- items destructurized from data
+        for (item of items.slice(10, 20)) {
```

Comment:
hmm pobierasz 250 filmów a następnie bierzesz stamtąd tylko 10 i to dokładnie nie wiesz jakich.
Spróbuj zrobić request który będzie pobierał dokładnie 10 filmów i to za każdym razem tych samych

Reply:
Możemy użyć w takim razie tego API: https://www.omdbapi.com/?s=dia&apikey=46e8c41c
które przy requestcie wyszukiwania, zwraca 10, zawsze takich samych tytułów.

Reply:
stwórz zatem tablicę z 10 id jakiś filmów, i będziemy robić 10 osobnych requestów. Możemy skorzystać z tego endpointa:

https://www.omdbapi.com/?i=tt0076759&apikey=46e8c41c